salt/edx/maintenance_tasks.sls

```diff
@@ -2,5 +2,5 @@ delete_edx_logs_older_than_30_days:
   cmd.run:
     - name: |
         find /edx/var/log -not -path "/edx/var/log/tracking/*" \
-        -type f -mtime +30 -name "*.gz" -o -name "lms-stderr.log.*" \
-        -exec rm -f {} \;
+        -type f \( -name "*.gz" -o -name "lms-stderr.log.*" \) \
```

Comment:
If you replace the `|` with `>-` then you can get rid of the `\` for the line continuations.